build command fails as of latest release

**Describe the bug**
The build command refers to something which cannot be found, on a brand new svelte kit project

**Logs**
```
 ant@apse  /tmp/wercel   npm run build                                                                                                         19:27:01 

> wercel@0.0.1 build /tmp/wercel
> svelte-kit build

Transforming...
  ⧗ client
  ⧗ server
Optimizing...
  ⧗ client
  ⧗ server
(node:1363910) UnhandledPromiseRejectionWarning: Error: Command failed: node /tmp/wercel/node_modules/snowpack/index.bin.js build --mount.src/routes=/_app/routes --mount.src/setup=/_app/setup --out=.svelte/build/unoptimized/server --ssr
/bin/sh: 1: node: not found

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:443:11)
    at Socket.emit (events.js:315:20)
    at Pipe.<anonymous> (net.js:674:12)
(node:1363910) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1363910) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
> Command failed: node /tmp/wercel/node_modules/snowpack/index.bin.js build --mount.src/routes=/_app/routes --mount.src/setup=/_app/setup --out=.svelte/build/unoptimized/client
/bin/sh: 1: node: not found

Error: Command failed: node /tmp/wercel/node_modules/snowpack/index.bin.js build --mount.src/routes=/_app/routes --mount.src/setup=/_app/setup --out=.svelte/build/unoptimized/client
/bin/sh: 1: node: not found

    at ChildProcess.exithandler (child_process.js:303:12)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Socket.<anonymous> (internal/child_process.js:443:11)
    at Socket.emit (events.js:315:20)
    at Pipe.<anonymous> (net.js:674:12)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wercel@0.0.1 build: `svelte-kit build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the wercel@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ant/.npm/_logs/2020-12-09T19_27_08_830Z-debug.log

```

**To Reproduce**
```
npm create svelte@next somedir
cd somedir
npm i
npm run build
```

--- a/packages/kit/src/api/build/index.js
+++ b/packages/kit/src/api/build/index.js
@@ -12,7 +12,7 @@ import { copy_assets } from '../utils';
 import { create_app } from '../../core/create_app';
 import { css_injection } from './css_injection';
 
-const exec = promisify(child_process.exec);
+const execFile = promisify(child_process.execFile);
 
 const snowpack_main = require.resolve('snowpack');
 const snowpack_pkg_file = path.join(snowpack_main, '../../package.json');
@@ -79,12 +79,11 @@ export async function build(config) {
 	const mount = [
 		`--mount.${config.files.routes}=/${config.appDir}/routes`,
 		`--mount.${config.files.setup}=/${config.appDir}/setup`
-	].join(' ');
+	];
 
 	const promises = {
-		transform_client: exec(`node ${snowpack_bin} build ${mount} --out=${UNOPTIMIZED}/client`, {
+		transform_client: execFile(process.argv[0], [snowpack_bin, 'build', ...mount, `--out=${UNOPTIMIZED}/client`], {
 			env: {
-				...process.env,
 				SVELTE_KIT_APP_DIR: config.appDir
 			}
 		}).then(
@@ -93,9 +92,8 @@ export async function build(config) {
 				render();
 			}
 		),
-		transform_server: exec(`node ${snowpack_bin} build ${mount} --out=${UNOPTIMIZED}/server --ssr`, {
+		transform_server: execFile(process.argv[0], [snowpack_bin, 'build', ...mount, `--out=${UNOPTIMIZED}/server`, '--ssr'], {
 			env: {
-				...process.env,
 				SVELTE_KIT_APP_DIR: config.appDir
 			}
 		}).then(() => {

